src/common/dsp/SurgeSuperOscillator.cpp

```diff
@@ -360,7 +355,7 @@ template <bool FM> void SurgeSuperOscillator::convolute(int voice, bool stereo)
    */
    unsigned int ipos;
 
-   if ((l_sync.v > 0) && (syncstate[voice] < oscstate[voice]))
+   if (syncstate[voice] < oscstate[voice])
```

Comment:
I'm pretty worried about this one too. Are you sure that ss[v] < os[v] is false when l_sync.v == 0?

Reply:
This one is actually fine, regtests don't seem to trip up over it. It's just S&H noise that is being tripped up (in absolute unison mode).

Reply:
And I think I found a proper solution for the issue in the end!